src/tr/merlinscans/src/eu/kanade/tachiyomi/extension/tr/merlinscans/MerlinScans.kt

```diff
@@ -1,12 +1,431 @@
+            description = document.select(".description-text").text()
+
+            author = document.select(".metadata-item .metadata-icon-label:contains(Yazar) + .metadata-value").text()
+
+            val statusText = document.select(".metadata-item .metadata-icon-label:contains(Durum) + .metadata-value").text()
+
+            status = when {
+                statusText.contains("Devam Ediyor", true) -> SManga.ONGOING
+                statusText.contains("Tamamlandı", true) -> SManga.COMPLETED
+                statusText.contains("Ara Verildi", true) -> SManga.ON_HIATUS
+                statusText.contains("İptal", true) -> SManga.CANCELLED
+                else -> SManga.UNKNOWN
+            }
+
+            genre = document.select(".category-badge")
+                .joinToString(", ") { it.text() }
+        }
+    }
+
+    override fun chapterListRequest(manga: SManga): Request {
+        return GET(baseUrl + manga.url, headers)
+    }
+
+    override fun chapterListSelector() = ".chapters-grid .chapter-item"
+
+    override fun chapterFromElement(element: Element): SChapter {
+        return SChapter.create().apply {
+            setUrlWithoutDomain(element.attr("href"))
+            name = element.select(".chapter-title").text()
+
+            val dateText = element.select(".chapter-date").text()
+            date_upload = parseDate(dateText)
+        }
+    }
+
+    private fun parseDate(dateStr: String): Long {
+        return when {
+            dateStr.contains("önce") -> {
+                val now = System.currentTimeMillis()
+                when {
+                    dateStr.contains("dakika") -> {
+                        val minutes = dateStr.filter { it.isDigit() }.toLongOrNull() ?: 0
+                        now - (minutes * 60 * 1000)
+                    }
+
+                    dateStr.contains("saat") -> {
+                        val hours = dateStr.filter { it.isDigit() }.toLongOrNull() ?: 0
+                        now - (hours * 60 * 60 * 1000)
+                    }
+
+                    dateStr.contains("gün") -> {
+                        val days = dateStr.filter { it.isDigit() }.toLongOrNull() ?: 0
+                        now - (days * 24 * 60 * 60 * 1000)
+                    }
+
+                    else -> now
+                }
+            }
+
+            else -> {
+                dateFormat.tryParse(dateStr) ?: alternativeDateFormat.tryParse(dateStr) ?: 0L
+            }
+        }
+    }
+
+    override fun chapterListParse(response: Response): List<SChapter> {
+        val document = response.asJsoup()
+        val allChapters = mutableListOf<SChapter>()
+        val originalUrl = response.request.url.toString()
+
+        val httpUrl = originalUrl.toHttpUrl()
+        val baseUrlBuilder = httpUrl.newBuilder()
+
+        // Tüm query parametrelerini kaldır
+        httpUrl.queryParameterNames.forEach { paramName ->
+            baseUrlBuilder.removeAllQueryParameters(paramName)
+        }
```

Comment:
Am I missing something or is it not possible to use `.query(null)` as in `normalizeUrl()`?